training/general/src/environments/tensorflow-2.8-cuda11/asset.yaml

```diff
@@ -1,10 +1,10 @@
-name: tensorflow-2.8-ubuntu20.04-py38-cuda11-gpu

```

Comment:
Will you please keep the line endings for this environment (multiple files) as-is? Too much looks changed when it was really just one line.

Reply:
should show correct changes now